### ISSUE TITLE
(suggestion) Remove sub directories

### ISSUE THREAD

**Issue by MortezaHeydari97** [sindresorhus/normalize-url#188]
Hi, 
Sometimes we need the pure domain only.
So I've a new suggestion to implement a new option to remove sub directories of the URL.

**Comment by sindresorhus** [comment]
Yeah, there could be a `removePath` option that default to `false`.

**Comment by MortezaHeydari97** [comment]
Yes, I'm agree.
Ok then, I will send the PR

**Comment by millette** [comment]
Related to #190 but I only wanted to remove certain sub directories.

**Comment by sindresorhus** [comment]
I think we could let it also accept a function that is provided the path components and is expected to return the new path components.
